### ISSUE TITLE
Having moreblocksmod enabled on a server results in the server crashing

### ISSUE THREAD

**Issue by Greg-J** [tvanprooyen/mbm#3]
MBM seems to work fine in single player, but crashes the dedicated server:

```
[15:58:23] [main/INFO]: Loading for game Minecraft 1.16.4
[15:58:23] [ForkJoinPool-1-worker-3/WARN]: The mod "autoconfig1u" contains invalid entries in its mod json:
- Unsupported root entry "$schema" at line 2 column 14
[15:58:23] [ForkJoinPool-1-worker-11/WARN]: The mod "libcd" contains invalid entries in its mod json:
- Unsupported root entry "licence" at line 9 column 12
[15:58:23] [ForkJoinPool-1-worker-13/WARN]: The mod "autoconfig1u" contains invalid entries in its mod json:
- Unsupported root entry "$schema" at line 2 column 14
[15:58:23] [ForkJoinPool-1-worker-10/WARN]: The mod "libcd" contains invalid entries in its mod json:
- Unsupported root entry "licence" at line 9 column 12
[15:58:23] [ForkJoinPool-1-worker-2/WARN]: The mod "autoconfig1u" contains invalid entries in its mod json:
- Unsupported root entry "$schema" at line 2 column 14
[15:58:23] [ForkJoinPool-1-worker-15/WARN]: The mod "autoconfig1u" contains invalid entries in its mod json:
- Unsupported root entry "$schema" at line 2 column 14
[15:58:23] [ForkJoinPool-1-worker-4/WARN]: The mod "cardinal-components-base" contains invalid entries in its mod json:
- Unsupported root entry "side" at line 8 column 11
[15:58:23] [ForkJoinPool-1-worker-2/WARN]: The mod "cardinal-components-entity" contains invalid entries in its mod json:
- Unsupported root entry "side" at line 8 column 11
[15:58:23] [ForkJoinPool-1-worker-3/WARN]: The mod "autoconfig1u" contains invalid entries in its mod json:
- Unsupported root entry "$schema" at line 2 column 14
[15:58:23] [ForkJoinPool-1-worker-11/WARN]: The mod "autoconfig1u" contains invalid entries in its mod json:
- Unsupported root entry "$schema" at line 2 column 14
[15:58:23] [ForkJoinPool-1-worker-2/WARN]: The mod "libcd" contains invalid entries in its mod json:
- Unsupported root entry "licence" at line 9 column 12
[15:58:23] [ForkJoinPool-1-worker-3/WARN]: The mod "autoconfig1u" contains invalid entries in its mod json:
- Unsupported root entry "$schema" at line 2 column 14
[15:58:23] [main/WARN]: Warnings were found! 
 - Mod 'LambDynamicLights' (lambdynlights) recommends version 1.14.0 or later of mod modmenu, which is missing!
	 - You must install version 1.14.0 or later of modmenu.
 - Conflicting versions found for fabric-networking-v0: used 0.1.10+6a2618f53a, also found 0.1.10+e00ecb5f95
 - Conflicting versions found for fabric-lifecycle-events-v1: used 1.2.0+6a2618f53a, also found 1.2.0+74cc3b2095
[15:58:23] [main/INFO]: [FabricLoader] Loading 146 mods: fallflyinglib@1.1.0, fabric-keybindings-v0@0.2.0+6a2618f53a, couplings@1.3.3, roughlyenoughitems-api@5.8.7, fabricloader@0.10.8, trinkets@2.6.7, org_apache_commons_commons-collections4@4.1, com_fasterxml_jackson_core_jackson-annotations@2.10.1, adapaxels@1.0.4, colorunchained@1.1.1-1.16.4, betternether@5.0.3, com_squareup_okio_okio@1.17.2, adventurez@1.1.4, fabric-game-rule-api-v1@1.0.3+a4467d2a3a, fabric-api-base@0.2.0+ba858fb83a, lambdynlights@1.3.2+1.16.4, adabraniummod@1.2.2, tweakeroo@0.10.0-dev.20201110.132827, decorative@3.0.0, org_jetbrains_kotlin_kotlin-stdlib@1.4.0, com_moandjiezana_toml_toml4j@+, colorfulcolumns@1.0.0, waila@1.9.22, com_neovisionaries_nv-websocket-client@2.10, playerabilitylib@1.2.1, fabric-blockrenderlayer-v1@1.1.4+6a2618f53a, simple_backpack@1.0.32, chat_heads@0.2.1+1.16.2, org_jetbrains_kotlin_kotlin-stdlib-jdk7@1.4.0, carpet-extra@1.4.16, fabric@0.26.0+1.16, moretotems@2.7.0, org_slf4j_slf4j-api@1.7.25, fdlink_common@0.7.14, fabric-networking-v0@0.1.10+6a2618f53a, fabric-screen-handler-api-v1@1.1.0+6a2618f53a, tk_valoeghese_zoesteriaconfig@1.3.2, greater_eye@1.1.3, columns@1.1.1, phosphor@0.6.0+build.7, launchersmod@2.1.4, org_jetbrains_kotlin_kotlin-stdlib-jdk8@1.4.0, fabric-tool-attribute-api-v1@1.2.5+6a2618f53a, roughlyenoughitems-runtime@5.8.7, midnight-hats@1.0.2, cloth-client-events-v0@1.4.5, fabric-crash-report-info-v1@0.1.2+b7f9825d3a, fabric-events-interaction-v0@0.4.1+6a2618f53a, mostructures@1.0.0-pre3+mc.1.16.2, net_dv8tion_jda@4.2.0_191, fabric-language-kotlin@1.4.0+build.1, origins@0.4.5, fabric-key-binding-api-v1@1.0.1+730711c63a, cloth-config2@4.8.3, org_jetbrains_kotlinx_kotlinx-coroutines-jdk8@1.3.9, fabric-resource-loader-v0@0.3.2+1944f3e03a, org_slf4j_slf4j-nop@1.7.25, malilib@0.10.0-dev.21+arne.2, org_jetbrains_kotlin_kotlin-reflect@1.4.0, worldedit@7.2.0+a51fa43, cardinal-components-entity@2.7.2, magna@1.4.2-1.16.2, staticdata@1.1.2, fabric-commands-v0@0.2.0+6a2618f53a, fabric-registry-sync-v0@0.6.1+6a2618f53a, adorn@1.12.1+1.16.4, cardinal-components-base@2.7.2, com_squareup_okhttp3_okhttp@3.13.0, fabric-mining-levels-v0@0.1.2+6a2618f53a, woodsplus@0.0.2, earthtojavamobs@1.2.2+1.16.4, fabric-item-groups-v0@0.2.1+6a2618f53a, fabric-models-v0@0.1.1+6a2618f53a, fdlink@0.7.14, shutupconsole@1.1, org_jetbrains_annotations@20.0.0, lithium-api@0.6.0, fabric-renderer-api-v1@0.3.3+6a2618f53a, linkedstorage@1.3.11-1.16, libcd@3.0.3+1.16.3, biomemakeover@1.16.4-1.1.3, connectiblechains@1.1.0, vanilla-hammers@3.0.1-1.16.3, cavebiomes@0.5.1, automated_crafting@1.3.13+MC1.16.2-1.16.4, fabric-structure-api-v1@1.1.0+f8ac1db23a, lithium@0.6.0, fabric-containers-v0@0.1.8+6a2618f53a, fabric-object-builder-api-v1@1.9.2+6a2618f53a, fdlink_1_15_2@0.7.14, ringsofascension@1.1.1, mcdw@1.1.8, waystones@1.0.9, fabric-rendering-data-attachment-v1@0.1.4+6a2618f53a, cloth-dynamic-registry-api-v1@1.4.8, fabric-textures-v0@1.0.5+6a2618f53a, worldeditcui@1.16.2+01, connected_block_textures@0.1.3+1.16, fabric-rendering-fluids-v1@0.1.12+6a2618f53a, reach-entity-attributes@1.0.1, allstackable@1.2.6, com_fasterxml_jackson_core_jackson-databind@2.10.1, fabric-lifecycle-events-v1@1.2.0+6a2618f53a, net_sf_trove4j_trove4j@3.0.3, fabric-renderer-registries-v1@2.2.0+6a2618f53a, com_vdurmont_emoji-java@5.1.1, fabric-loot-tables-v1@1.0.1+6a2618f53a, com_fasterxml_jackson_core_jackson-core@2.10.1, carpet@1.4.17, moreblocksmod@1.0.0, fabric-particles-v1@0.2.2+6a2618f53a, slotlink@2.1.1+1.16.4, minecraft@1.16.4, roughlyenoughitems-default-plugin@5.8.7, fabric-networking-blockentity-v0@0.2.5+6a2618f53a, jankson@3.0.1+j1.2.0, cloth-basic-math@0.5.1, fabric-command-api-v1@1.0.9+6a2618f53a, cotton-resources@1.7.4, static-content@1.0.1-1.16.2, origins-classes@1.1.0, fabric-renderer-indigo@0.4.3+6a2618f53a, mixedslab@1.16.3-1.2, roughlyenoughitems@5.8.7, fabric-item-api-v1@1.2.0+6a2618f53a, libgui@3.2.1+1.16.3, fabric-rendering-v0@1.1.1+6a2618f53a, fabric-rendering-v1@1.4.0+6a2618f53a, spruceui@1.6.4, autoconfig1u@3.3.1, winged@1.16.2-1.4.1, org_json_json@20170516, fabric-content-registries-v0@0.1.9+6a2618f53a, fabric-tag-extensions-v0@1.0.3+6a2618f53a, fastfurnace@2.3, fabric-biome-api-v1@3.1.0+2e23b97c3a, hplus@1.0.7, itemscroller@0.15.0-dev.20201103.183933, byg@1.1.3, zoob@2.5.0, fabric-events-lifecycle-v0@0.2.0+6a2618f53a, libstructure@1.3, vanillaautomated@1.1.5, betterend@0.6.2-beta, fdlink_1_16@0.7.14, fabric-object-builders-v0@0.7.1+6a2618f53a
[15:58:23] [main/WARN]: Mod `com_moandjiezana_toml_toml4j` (+) does not respect SemVer - comparison support is limited.
[15:58:23] [main/WARN]: Mod `net_dv8tion_jda` (4.2.0_191) does not respect SemVer - comparison support is limited.
[15:58:24] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.2 Source=file:/C:/Users/Greg/AppData/Roaming/.minecraft/afterdark/fabric-server-launch.jar Service=Knot/Fabric Env=SERVER
[15:58:24] [main/WARN]: Reference map 'hopper-plus-fabric-1.16.4-refmap.json' for hplus.mixins.json could not be read. If this is a development environment you can ignore this message
[15:58:26] [main/WARN]: Reference map 'woods-plus-refmap.json' for woodsplus.mixins.json could not be read[15:58:33] [main/INFO]: [BetterNether] Enabled Vanilla Hammers Integration
[15:58:35] [main/INFO]: Discovered plugin waila:core provided by waila at mcp.mobius.waila.addons.core.PluginCore
[15:58:35] [main/INFO]: Discovered plugin waila:vanilla provided by waila at mcp.mobius.waila.addons.minecraft.PluginMinecraft
[15:58:35] [main/INFO]: Registering plugin at mcp.mobius.waila.addons.core.PluginCore
[15:58:35] [main/INFO]: Registering plugin at mcp.mobius.waila.addons.minecraft.PluginMinecraft
[15:58:35] [main/INFO]: Origins is initializing. Have fun!
[15:58:35] [main/INFO]: WorldEdit for Fabric (version 7.2.0+a51fa43) is loaded
[15:58:37] [main/INFO]: [Earth2Java] Mod loaded! Enjoy :D
[15:58:37] [main/INFO]: Loaded 14 Static Content files for vanilla-hammers:hammers
[15:58:37] [main/INFO]: [All Stackable] Start loading!
[15:58:37] [main/INFO]: [All Stackable] Command registered.
[15:58:37] [main/INFO]: ignored "badasintended.slotlink.compat.StockpileCompat" from "slotlink"
[15:58:38] [main/INFO]: BYG: Blocks registered!
[15:58:38] [main/INFO]: BYG: Items registered!
[15:58:38] [main/INFO]: BYG: Entities registered!
[15:58:39] [main/INFO]: Adding Numerical Biome ID's
[15:58:39] [main/INFO]: Added Numerical Biome ID's!
[15:58:39] [main/INFO]: BYG: Biomes registered!
[15:58:39] [main/INFO]: BYG: Decorators registered!
[15:58:39] [main/INFO]: BYG: Features registered!
[15:58:39] [main/INFO]: BYG: Surface builders Registered!
[15:58:39] [main/INFO]: Reading json
[15:58:39] [main/INFO]: Reading Sub-Biomes json
[15:58:39] [main/INFO]: BYG: "Common Setup" Event Complete!
[15:58:39] [main/INFO]: BYG: Compostable items added!
[15:58:39] [main/INFO]: BYG: Added Hoeable Blocks!
[15:58:39] [main/INFO]: BYG: Added Flammables!
[15:58:39] [main/INFO]: BYG: Added strippable Blocks...
[15:58:39] [main/INFO]: BYG: "Load Complete" Event Complete!
[15:58:39] [main/FATAL]: Failed to start the minecraft server
java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'moreblocksmod'!
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:53) ~[fabric-server-launch.jar:?]
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke(EntrypointUtils.java:36) ~[fabric-server-launch.jar:?]
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointServer.start(EntrypointServer.java:32) ~[fabric-server-launch.jar:?]
	at net.minecraft.server.Main.main(Main.java:92) [intermediary-server.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_261]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_261]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_261]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_261]
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:224) [fabric-server-launch.jar:?]
	at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:141) [fabric-server-launch.jar:?]
	at net.fabricmc.loader.launch.knot.KnotServer.main(KnotServer.java:27) [fabric-server-launch.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_261]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_261]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_261]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_261]
	at net.fabricmc.loader.launch.server.FabricServerLauncher.launch(FabricServerLauncher.java:62) [fabric-server-launch.jar:?]
	at net.fabricmc.loader.launch.server.FabricServerLauncher.setup(FabricServerLauncher.java:106) [fabric-server-launch.jar:?]
	at net.fabricmc.loader.launch.server.FabricServerLauncher.main(FabricServerLauncher.java:50) [fabric-server-launch.jar:?]
Caused by: java.lang.NoClassDefFoundError: net/fabricmc/fabric/api/client/render/fluid/v1/FluidRenderHandler
	at com.tylervp.moreblocksmod.onInitialize(moreblocksmod.java:359) ~[moreblocksmod-0.1.1.0.jar:?]
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:50) ~[fabric-server-launch.jar:?]
	... 17 more
Caused by: java.lang.ClassNotFoundException: net.fabricmc.fabric.api.client.render.fluid.v1.FluidRenderHandler
	at java.net.URLClassLoader.findClass(Unknown Source) ~[?:1.8.0_261]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.8.0_261]
	at sun.misc.Launcher$AppClassLoader.loadClass(Unknown Source) ~[?:1.8.0_261]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.8.0_261]
	at net.fabricmc.loader.launch.server.InjectingURLClassLoader.loadClass(InjectingURLClassLoader.java:56) ~[fabric-server-launch.jar:?]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.8.0_261]
	at net.fabricmc.loader.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:168) ~[fabric-server-launch.jar:?]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.8.0_261]
	at com.tylervp.moreblocksmod.onInitialize(moreblocksmod.java:359) ~[moreblocksmod-0.1.1.0.jar:?]
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:50) ~[fabric-server-launch.jar:?]
	... 17 more

```

**Comment by RDKRACZ** [comment]
Crashes for me too. Newest fabric loader and api

**Comment by tvanprooyen** [comment]
This issue is corrected for MBM version 0.1.2.0. This will be released in the next two weeks as more things are being added.

**Comment by tvanprooyen** [comment]
The update 0.1.2.0 is out. Please let me know if there is any other issues. Thanks!
